perception/autoware_tensorrt_bevdet/src/bevdet_node.cpp

```diff
@@ -0,0 +1,247 @@
+// Copyright 2024 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#include "autoware/tensorrt_bevdet/bevdet_node.hpp"
+
+using Label = autoware_perception_msgs::msg::ObjectClassification;
+std::map<int, std::vector<int>> colormap{
+  {0, {0, 0, 255}},  // dodger blue
+  {1, {0, 201, 87}}, {2, {0, 201, 87}}, {3, {160, 32, 240}}, {4, {3, 168, 158}}, {5, {255, 0, 0}},
+  {6, {255, 97, 0}}, {7, {30, 0, 255}}, {8, {255, 0, 0}},    {9, {0, 0, 255}},   {10, {0, 0, 0}}};
+
+uint8_t getSemanticType(const std::string & class_name)
+{
+  if (class_name == "car") {
+    return Label::CAR;
+  } else if (class_name == "truck") {
+    return Label::TRUCK;
+  } else if (class_name == "bus") {
+    return Label::BUS;
+  } else if (class_name == "trailer") {
+    return Label::TRAILER;
+  } else if (class_name == "bicycle") {
+    return Label::BICYCLE;
+  } else if (class_name == "motorcycle") {
+    return Label::MOTORCYCLE;
+  } else if (class_name == "pedestrian") {
+    return Label::PEDESTRIAN;
+  } else {
+    return Label::UNKNOWN;
+  }
+}
+
+void Getinfo(void)
+{
+  cudaDeviceProp prop;
+
+  int count = 0;
+  cudaGetDeviceCount(&count);
+  printf("\nGPU has cuda devices: %d\n", count);
```

Comment:
@cyn-liu 

Thank you for your PR!
Please use `RCLCPP_INFO_THROTTLE` instead of `printf`.